examples/angular-cli/src/stories/addon-jest.stories.ts

```diff
@@ -2,7 +2,7 @@ import { storiesOf } from '@storybook/angular';
 import { withTests } from '@storybook/addon-jest';
 
 import { AppComponent } from '../app/app.component';
-// eslint-disable-next-line import/no-unresolved
+// eslint-disable-next-line
```

Comment:
Why?

BTW precommit hooks fails here when I try to merge next into my branch

Reply:
Ow I remember:

Usually this file does NOT exist.. It's a generated file not checked into source-control. We generate it just before we start|build the storybook.

This rule ignores the fact that it's not there.

BUT when it IS there, there's a rule warning us that there's an eslint rule being ignored that isn't being reported. (because now the file is present).

I don't know a good fix for this. Sometimes the file is there, sometimes it isn't. Either cases, we don't want eslint to complain about it.

Would you know a good way of doing that?

Reply:
Fixed with https://github.com/storybookjs/storybook/pull/8289/files#diff-46e956f0103273a39dd3c85b4093053a

Reply:
But I meant, why the wildcard disable instead of particular rules

Reply:
In the hopes that eslint would then not complain in either case, But I guess it did.